src/streams/EditStreamCard.js

```diff
@@ -5,45 +5,96 @@ import { View } from 'react-native';
 
 import Input from '../common/Input';
 import ZulipTextIntl from '../common/ZulipTextIntl';
-import SwitchRow from '../common/SwitchRow';
 import ZulipButton from '../common/ZulipButton';
-import styles, { createStyleSheet } from '../styles';
-import { IconPrivate } from '../common/Icons';
+import styles from '../styles';
+import SelectableOptionRow from '../common/SelectableOptionRow';
 
-const componentStyles = createStyleSheet({
-  switchRow: {
-    paddingLeft: 8,
-    paddingRight: 8,
-  },
-});
+type AccessPolicy =
+  | 'public'
+  | 'web_public'
+  | 'private_shared_history'
+  | 'private_protected_history';
+
+type StreamPolicySettings = {
+  // The properties on a stream object that are related to the access policy.
```

Comment:
nit: This comment is describing the type as a whole, and in fact is describing its interface: the information that a reader will want to have when using this type elsewhere, not only when making changes to its internals.

For information about something's interface, use jsdoc:
```suggestion
/** The properties on a stream object that are related to the access policy. */
type StreamPolicySettings = {
```

(Hmm, I should put that in the style guide.)